src/cargo/util/job.rs

```diff
@@ -42,21 +42,21 @@ mod imp {
 
 #[cfg(windows)]
 mod imp {
-    extern crate kernel32;
     extern crate winapi;
-    extern crate psapi;
 
     use std::ffi::OsString;
     use std::io;
     use std::mem;
     use std::os::windows::prelude::*;
+    use self::winapi::shared::{basetsd, ntdef, minwindef};
+    use self::winapi::um::{handleapi, jobapi, jobapi2, processthreadsapi, psapi, synchapi, winbase, winnt};
```

Comment:
Could this perhaps just use `*` below? There's no hope of me ever knowing how this list is generated :(

Reply:
Yeah sure, there shouldn't be any future conflicts since it's limited by the enabled features (I hope it stays that way).
That basically was just the explicit safe route, depending on what modules are used.